lib/utils/pre-checks.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+const fs = require('fs-extra');
+const os = require('os');
+const path = require('path');
+const semver = require('semver');
+const latestVersion = require('latest-version');
+const pkg = require('../../package.json');
+
+/**
+ * Checks if a version update is available
+ * @param {UI} ui ui instance
+ * @param {System} system System instance
+ */
+module.exports = function preChecks(ui, system) {
+    const tasks = [{
+        title: 'Checking for Ghost-CLI updates',
+        task: () => latestVersion(pkg.name).then((latest) => {
+            if (semver.lt(pkg.version, latest)) {
+                const chalk = require('chalk');
+
+                ui.log(
+                    'You are running an outdated version of Ghost-CLI.\n' +
+                    'It is recommended that you upgrade before continuing.\n' +
+                    `Run ${chalk.cyan('`npm install -g ghost-cli@latest`')} to upgrade.\n`,
+                    'yellow'
+                );
+            }
+        })
+    }, {
+        title: 'Checking ~/.config folder ownership',
```

Comment:
I think it might be better to use something like `Ensuring` instead of Checking - I wouldn't expect changes to be made with checking